indigo_app/templates/indigo_api/_work_form_content.html

```diff
@@ -0,0 +1,498 @@
+{% load i18n %}
+
+{% block work-content %}
+  {% if work.pk and perms.indigo_api.change_work or not work.pk and perms.indigo_api.add_work %}
+  {% else %}
+    <div class="alert alert-danger">{% trans "You don't have permission to create or change a work." %}</div>
+  {% endif %}
+
+  {% if form.errors %}
+    <div class="alert alert-danger" role="alert">
+      <p>{% trans "There were some errors in the information you entered. Please correct the following:" %}</p>
+      {{ form.non_field_errors }}
+      <ul>
+        {% for field in form %}
+          {% if field.errors %}<li>{{ field.label }}: {{ field.errors|striptags }}</li>{% endif %}
+        {% endfor %}
+      </ul>
+    </div>
+  {% endif %}
+
+  <form
+      id="edit-work-form"
+      method="POST"
+      enctype="multipart/form-data"
+      data-component="WorkForm"
+      {% if "HTTP_HX_REQUEST" in request.META %}
```

Comment:
You can check for `request.htmx` - https://django-htmx.readthedocs.io/en/latest/middleware.html